### DISCUSSION TITLE
关于claude-v1-100k 福利！

### DISCUSSION THREAD

**Discussion by q2408808** [viopsa233/AI-testers-Forum#59] [交流]
我在官网申请以后，在网页版试用，发现真的是claude-v1-100k，我用了发了3万多字的小说章节进行试验，确实可以总结加续写。就是反应稍微要个几十秒，不过已经算很好了。另外它网页版一次输出的长度看起来不会超过1000字，我让它用2000字总结，还是8,900字就结束了。目前来说限制比poe的要少，且不是poe的阉割版，我用poe总结，还说有血腥暴力内容，明明只是生化危机而已。有长文章内容需要总结的，可以看一下这个。以下是截图：
![1684552521269(1)](https://github.com/viopsa233/AI-testers-Forum/assets/128444814/8cabf15c-6590-4574-b361-7453cbe5189f)
![1684552530125](https://github.com/viopsa233/AI-testers-Forum/assets/128444814/a5e061b7-e35f-4f1e-afa8-b85ced67e9f2)
![1684552544797(1)](https://github.com/viopsa233/AI-testers-Forum/assets/128444814/c386ebe6-b8db-46d4-9b8b-edf551edb082)
![1684552724528](https://github.com/viopsa233/AI-testers-Forum/assets/128444814/2a11f0cd-930d-4187-866f-3261b6d5d552)

**Comment by q2408808** [comment]
逻辑上也不算差了，何况上下文长度属于碾压级别。
![1684553805647](https://github.com/viopsa233/AI-testers-Forum/assets/128444814/9354c749-3c1b-45a4-9379-e7e16130e3ea)

**Comment by viopsa233** [comment]
有无链接

**Reply by q2408808** [reply]
https://www.anthropic.com/product 我申请了两次，编了俩公司名字，俩都过了。这个应该没什么限制。完全免费

**Reply by viopsa233** [reply]
> https://www.anthropic.com/product 我申请了两次，编了俩公司名字，俩都过了。这个应该没什么限制。完全免费

怪不得,我之前都是申请个人的

**Reply by viopsa233** [reply]
> https://www.anthropic.com/product 我申请了两次，编了俩公司名字，俩都过了。这个应该没什么限制。完全免费

你申请几天过?

**Reply by q2408808** [reply]
两天吧，这玩意毫无热度。感觉都没人用。界面上也没有模组信息，要不是我看后台，都不知道是100 k

**Reply by viopsa233** [reply]
> 两天吧，这玩意毫无热度。感觉都没人用。界面上也没有模组信息，要不是我看后台，都不知道是100 k

都在用poe呢,主要是因为个人申请困难

**Reply by q2408808** [reply]
跟open ai一样，api收费，官网免费。我看着是这样的。看看大佬们能不能用什么办法给它的cookie啥的也弄一下，这100k的字数玩啥都能玩到爽。

**Reply by q2408808** [reply]
它分成正常的Claude100k和即时版100k，api里有介绍。poe 那个是即时版的，应该还是要笨一点

**Reply by ABC20232001** [reply]
> https://www.anthropic.com/product 我申请了两次，编了俩公司名字，俩都过了。这个应该没什么限制。完全免费

怎么知道申请是否通过呢？还是同样的链接点进去吗？

**Reply by dot307** [reply]
啊，那公司网址怎么编的？

**Reply by viopsa233** [reply]
> 啊，那公司网址怎么编的？

公司网站非必填啊

**Reply by dot307** [reply]
嗷好，邮箱呢，国内邮箱可以吗

**Reply by viopsa233** [reply]
> 嗷好，邮箱呢，国内邮箱可以吗

不知道,建议用国外的

**Reply by dot307** [reply]
好谢谢

**Reply by q2408808** [reply]
我微软和qq邮箱各成功了一次

**Reply by q2408808** [reply]
> > https://www.anthropic.com/product 我申请了两次，编了俩公司名字，俩都过了。这个应该没什么限制。完全免费
> 
> 怎么知道申请是否通过呢？还是同样的链接点进去吗？
邮箱会收到专属链接。

**Reply by dot307** [reply]
> > > https://www.anthropic.com/product 我申请了两次，编了俩公司名字，俩都过了。这个应该没什么限制。完全免费
> > 
> > 
> > 怎么知道申请是否通过呢？还是同样的链接点进去吗？
> > 邮箱会收到专属链接。

哇。花了多久呀。。我之前那个都一个多月了还没，前天又重新用公司申了一遍

**Comment by araea** [comment]
支持！！！

**Comment by vifdiwxfs** [comment]
推一个网站
[nat.dev](url)
这里面都有gpt4和claude-v1-100k不过都是按token➕20%收费的

**Comment by ghost** [comment]
如果用API的话能绕过道德限制吗？比如给AI的回复先加一个免责声明让她从免责声明开始补全

**Reply by q2408808** [reply]
我没有尝试过API，但是免责申明其实在官网就可以绕过。它的官网是提供直接编辑对话的，我可以将claude的对话从 因为道德和法律，我无法告诉你，直接改成, 现在我将会告诉你，然后点一下generate，它就给出答案了。

**Comment by ghost** [comment]
因为Claude使用的词库和GPT不一样，所以token计算也是不一样的
Claude的token计算器 https://github.com/anthropics/anthropic-sdk-python/blob/main/anthropic/tokenizer.py

**Reply by q2408808** [reply]
是不一样，像我图里能看到，同样的字数claude官方是3万4，而同样的内容，QT那个用户输入已经4万了，似乎claude还要更省token。

**Comment by cool-delete** [comment]
官网是哪里可以generate呢?

**Reply by q2408808** [reply]
edit chat 里 可以随便修改，修改完以后右下角有个generate，会根据你写的内容再生成一小段，然后就可以切回正常chat了

**Reply by cool-delete** [reply]
我去https://www.anthropic.com/product 网址里面只有叫我去slack尝试?
slack没有edit chat吧 是我误解了什么嘛

**Reply by q2408808** [reply]
申请，request acess.不过这里也没人反馈。可能确实很难通过吧。。通过了会收到一个邮件，给你一个专属于你邮箱的链接

**Reply by viopsa233** [reply]
> 申请，request acess.不过这里也没人反馈。可能确实很难通过吧。。通过了会收到一个邮件，给你一个专属于你邮箱的链接

确实难通过,我3个号没有一个通过😥

**Reply by ufomonb** [reply]
> > 申请，request acess.不过这里也没人反馈。可能确实很难通过吧。。通过了会收到一个邮件，给你一个专属于你邮箱的链接
> 
> 确实难通过,我3个号没有一个通过😥

公司和公司网址随便填？

**Reply by q2408808** [reply]
> > > 申请，request acess.不过这里也没人反馈。可能确实很难通过吧。。通过了会收到一个邮件，给你一个专属于你邮箱的链接
> > 
> > 
> > 确实难通过,我3个号没有一个通过😥
> 
> 公司和公司网址随便填？

我好像是企查查里随便搜的
